packages/core/src/delegation/Delegation.utils.ts

```diff
@@ -0,0 +1,23 @@
+import type { IDelegationBaseNode } from '@kiltprotocol/types'
+import { SDKErrors, DataUtils } from '@kiltprotocol/utils'
+import { isHex } from '@polkadot/util'
+
+export default function errorCheck(
+  id: IDelegationBaseNode['id'],
+  account: IDelegationBaseNode['account'],
+  revoked: IDelegationBaseNode['revoked']
+): void {
+  if (!id || typeof id !== 'string') {
+    throw SDKErrors.ERROR_NODE_QUERY(id)
```

Comment:
seems like an odd choice to signal a missing id ?

Reply:
Yes, I have updated to be in line with what is happening.